terraform/modules/windows-test-domain/scripts/WEC/join_domain.ps1

```diff
@@ -1,7 +1,6 @@
-$User = "$env:domain\wecserver"
-$Password = ConvertTo-SecureString -String "Edhellen$" -AsPlainText -Force
+$User = "SHIRE\wecserver"
```

Comment:
should we not change this from SHIRE ?

Reply:
We should but I'll raise that as a separate issue. It doesn't cause a conflict so it's fine for now.

Reply:
I think there are strict rules about what a NETBIOS name can be (chars and length) so I don't think it can just be the first part of the domain path.

Reply:
right, fair enough!

Reply:
https://github.com/alphagov/cyber-security-windows-sandbox/issues/23